packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -276,15 +274,15 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
 
   @override
   Widget build(BuildContext context) {
-    final Color resolvedBackgroundColor = CupertinoDynamicColor.resolve(widget.backgroundColor, context);
+    final Color? resolvedBackgroundColor = CupertinoDynamicColor.resolve(widget.backgroundColor, context);
```

Comment:
here the code assumes `resolve` may return null, but on line 256 it assumes it will return non-null

Reply:
```
const Color _kHighlighterBorder = CupertinoDynamicColor.withBrightness(
  color: Color(0x33000000),
  darkColor: Color(0x33FFFFFF),
);
```

Completely specifies _kHighlighterBorder's color properties so I think line 256 can correctly assume that resolve's return value is non-null.

Reply:
Here it is non-null because it is used in a non-null context (`BoxDecoration(color: resolvedBackgroundColor)` L308). L256 required a null check because it is used in a non-null context (`BorderSide(width: 0.0, color: resolvedBorderColor)`)